update.py

```diff
@@ -23,13 +23,21 @@
 from common import AV_DEFINITION_S3_PREFIX
 from common import CLAMAVLIB_PATH
 from common import get_timestamp
-
+import shutil
 
 def lambda_handler(event, context):
     s3 = boto3.resource("s3")
     s3_client = boto3.client("s3")
 
     print("Script starting at %s\n" % (get_timestamp()))
+
+    #clean /tmp on the ec2 instance because it doesn't clean itself
+    for root, dirs, files in os.walk(AV_DEFINITION_PATH):
```

Comment:
Do you need to walk yourself? Or does shutil.rmtree do that for you?

Reply:
Updated to just nuke the entire folder so anything under /tmp/